base/loading.jl

```diff
@@ -419,8 +420,9 @@ function require(mod::Symbol)
             cachefile = compilecache(mod)
             m = _require_from_serialized(1, mod, cachefile, last)
             if isa(m, Exception)
-                warn("Compilecache failed to create a usable precompiled cache file for module $name. Got:")
+                warn("`compilecache` failed to create a usable precompiled cache file for module $name. Got:")
```

Comment:
I'm using it in it's typographic form for its similarity to quotation marks, used to indicate that the "word" is special. I believe Markdown inherited this usage, not invented it. Stylistically, I usually use the " ` " for code and ' " ' for other text as I think it provides an additional hint to the reader why that word is special.